README.md

```diff
@@ -51,8 +52,16 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
+
 // write code here
+var pizza = {
+  temperature: 70,
+  bake: function (){
+    this.temperature = 300
+  }
+}
 ```
+-->
```

Comment:
I accidentally messed up the formatting of the document. They don't mean anything though